background.js

```diff
@@ -1,9 +1,16 @@
 const oldReddit = "https://old.reddit.com";
+var disabled = false;
 
+// Redirect requests
 chrome.webRequest.onBeforeRequest.addListener(
   function(details) {
+    // Check if disabled
+    if(disabled)  {
+      return;
+    }
+
     // Exclude poll pages
-    if (details.url.match(/^https?:\/\/(www\.)*reddit.com\/poll/)) {
+    if (/^https?:\/\/(www\.)*reddit.com\/poll/.test(details.url)) {
```

Comment:
Since the regex is just used to see if there is a match, I thought test() would be a more direct way to look for matches when the result is not required.
I can undo that change if you would like.

Reply:
Note that `.test` is faster when doing boolean tests, so it's probably worth it.

[source](https://stackoverflow.com/questions/10940137/regex-test-v-s-string-match-to-know-if-a-string-matches-a-regular-expression)

Reply:
So, should this be reverted to use ```.match()``` or should we keep the change to ```.test()```?

Reply:
Just leave it as `.test` :)